--- bluez-4.47/src/oui.c~	2009-01-08 17:00:12.000000000 +0100
+++ bluez-4.47/src/oui.c	2009-08-08 13:51:09.519278151 +0200
@@ -38,7 +38,7 @@
 
 /* http://standards.ieee.org/regauth/oui/oui.txt */
 
-#define OUIFILE "/var/lib/misc/oui.txt"
+#define OUIFILE "/usr/share/hwdata/oui.txt"
 
 char *ouitocomp(const char *oui)
 {
@@ -48,12 +48,7 @@
 
 	fd = open("oui.txt", O_RDONLY);
 	if (fd < 0) {
-		fd = open(OUIFILE, O_RDONLY);
-		if (fd < 0) {
-			fd = open("/usr/share/misc/oui.txt", O_RDONLY);
-			if (fd < 0)
-				return NULL;
-		}
+		return NULL;
 	}
 
 	if (fstat(fd, &st) < 0) {
