--- bluez-4.53/Makefile.am~	2009-09-03 05:58:36.000000000 +0200
+++ bluez-4.53/Makefile.am	2009-09-22 12:43:34.459336083 +0200
@@ -194,7 +194,7 @@
 src_bluetoothd_LDADD = lib/libbluetooth.la @GLIB_LIBS@ @DBUS_LIBS@ -ldl
 src_bluetoothd_LDFLAGS = -Wl,--export-dynamic \
 					-Wl,--version-script=src/bluetooth.ver
-src_bluetoothd_DEPENDENCIES = src/bluetooth.ver
+src_bluetoothd_DEPENDENCIES = src/bluetooth.ver lib/libbluetooth.la
 
 builtin_files = src/builtin.h $(builtin_nodist)
 
