--- bluez-4.78/src/event.c.orig	2010-11-08 11:58:02.359781937 +0200
+++ bluez-4.78/src/event.c	2010-11-08 11:58:34.456448604 +0200
@@ -489,8 +489,16 @@
 
 		/* It's ok to cast end between const and non-const since
 		 * we know it points to inside of name which is non-const */
-		if (!g_utf8_validate(name, -1, (const char **) &end))
-			*end = '\0';
+		if (!g_utf8_validate(name, -1, (const char **) &end)) {
+			char *utf8_name;
+
+			utf8_name = g_convert(name, -1, "UTF-8", "ISO-8859-1", NULL, NULL, NULL);
+			if (utf8_name) {
+				memcpy(name, utf8_name, 248);
+				g_free(utf8_name);
+			} else
+				*end = '\0';
+		}
 
 		write_device_name(local, peer, name);
 	}
