diff -Nru bluez-4.53/tools/parser.y.orig bluez-4.53/tools/parser.y
--- bluez-4.53/tools/parser.y.orig	Thu Oct 31 10:09:48 2002
+++ bluez-4.53/tools/parser.y	Sun Mar 23 18:18:14 2003
@@ -147,7 +147,11 @@
 	if (filename) {
 		snprintf(file, MAXPATHLEN,  "%s", filename);
 	} else {
-		snprintf(file, MAXPATHLEN, "%s/.bluetooth/rfcomm.conf", getenv("HOME"));
+		char *etc_dir = getenv("HOME_ETC");
+		if (etc_dir)
+			snprintf(file, MAXPATHLEN, "%s/.bluetooth/rfcomm.conf", etc_dir);
+		else
+			snprintf(file, MAXPATHLEN, "%s/.bluetooth/rfcomm.conf", getenv("HOME"));
 
 		if ((getuid() == 0) || (access(file, R_OK) < 0))
 			snprintf(file, MAXPATHLEN, "/etc/bluetooth/rfcomm.conf");
