--- bluez-5.61/profiles/audio/media.c.orig	2021-07-07 15:45:55.000000000 +0200
+++ bluez-5.61/profiles/audio/media.c	2021-08-23 11:36:23.090317414 +0200
@@ -1281,7 +1281,7 @@
 	return media_player_send(mp, "Stop");
 }
 
-static bool pause(void *user_data)
+static bool bluez_media_pause(void *user_data)
 {
 	struct media_player *mp = user_data;
 
@@ -1331,7 +1331,7 @@
 	.set_volume = set_volume,
 	.play = play,
 	.stop = stop,
-	.pause = pause,
+	.pause = bluez_media_pause,
 	.next = next,
 	.previous = previous,
 };
