--- bluez-4.99/tools/hid2hci.c~	2011-05-03 10:20:36.000000000 +0200
+++ bluez-4.99/tools/hid2hci.c	2012-06-02 17:30:34.286326842 +0200
@@ -291,7 +291,7 @@
 	if (udev == NULL)
 		goto exit;
 
-	snprintf(syspath, sizeof(syspath), "%s/%s", udev_get_sys_path(udev), devpath);
+	snprintf(syspath, sizeof(syspath), "/sys/%s", devpath);
 	udev_dev = udev_device_new_from_syspath(udev, syspath);
 	if (udev_dev == NULL) {
 		fprintf(stderr, "error: could not find '%s'\n", devpath);
